modules/local/exomedepth/merge_count/main.nf

```diff
@@ -0,0 +1,22 @@
+// MERGE COUNT FILES
+process EXOMEDEPTH_COUNT_MERGE {
+    publishDir "$params.outdir/exomedepth/counts", mode: 'copy'
+
+    input:
+    tuple val(meta), path(files)
+
+    output:
+    tuple val(meta), path("${meta.id}_${meta.chr}.txt")
+
+    script:
+    """
+    for file in $files; do
+        if [ -f ${meta.id}_${meta.chr}.txt ]; then
+            paste ${meta.id}_${meta.chr}.txt <(awk '{print \$5}' \$file) > temp_auto.txt
+            mv temp_auto.txt ${meta.id}_${meta.chr}.txt
+            else
+            cp \$file ${meta.id}_${meta.chr}.txt
+        fi
+    done
+    """
```

Comment:
```suggestion
    output:
    tuple val(meta), path("${prefix}.txt")

    script:
    prefix = task.ext.prefix ?: "${meta.id}_${meta.chr}"
    """
    for file in $files; do
        if [ -f ${prefix}.txt ]; then
            paste ${prefix}.txt <(awk '{print \$5}' \$file) > temp_auto.txt
            mv temp_auto.txt ${prefix}.txt
            else
            cp \$file ${prefix}.txt
        fi
    done
    """
```
A small change to make it possible to configure the filename via the the configuration file. This makes it easy to tweak the process without having to go into the code itself.

Reply:
You can set the prefix in a configuration file like this:
```groovy
process {
    withName: EXOMEDEPTH_COUNT_MERGE {
        ext.prefix = ...
    }
}
```